src/domain/usecases/GetDataFormsForGenerationUseCase.ts

```diff
@@ -67,7 +67,12 @@ export class GetDataFormsForGenerationUseCase {
                 return _(dataFormsSpecificCustom)
                     .concat(dataFormsDefaultCustom)
                     .concat(settings.showAutogeneratedForms ? [dataFormAutogenerated] : [])
-                    .filter(dataForm_ => settings.isDataFormVisibleForCurrentUser(dataForm_.id))
+                    .filter(dataForm_ => {
+                        const isAutoGenerated = dataForm_.templateId === dataFormAutogenerated.templateId;
```

Comment:
`dataForm === dataFormAutogenerated` should also work, but comparing IDs is ok too.